app/controllers/backend/base_controller.rb

```diff
@@ -32,6 +32,7 @@ class BaseController < ::BaseController
     before_action :set_current_period_interval
     before_action :set_current_period
     before_action :publish_backend_action
+    before_action :first_run_or_backend
```

Comment:
This can't be in the core of Ekylibre, as all Ekylibre install won't have the Hajimari plugin.